packages/sqs/lib/utils/sqsUtils.ts

```diff
@@ -219,3 +224,12 @@ export async function deleteQueue(
     console.log(`Failed to delete: ${err.message}`)
   }
 }
+
+export function calculateOutgoingMessageSize(message: unknown) {
+  const messageBody = JSON.stringify(message)
+  return calculateSqsMessageBodySize(messageBody)
+}
+
+export function calculateSqsMessageBodySize(messageBody: SendMessageCommandInput['MessageBody']) {
```

Comment:
what's SQS-specific to this?

Reply:
Only the SQS layer is aware of how the message body is represented (serialized) in the message and what kind of extra attributes may be present that will impact the resulting message size.

Reply:
can you add a comment with explanation how it will need to be changed in case we'll start using attributes there?